README.md

```diff
@@ -312,6 +314,20 @@ Note that in `development` and `test` modes the file isn't uploaded. Instead, it
 Rails.application.config.app_profiler.middleware_action = AppProfiler::Middleware::UploadAction
 ```
 
+## Profiler backends
+
+It is possible to configure app_profiler to use the [`vernier`](https://github.com/jhawthorn/vernier) or stackprof. To use vernier, it must be added separately in the application Gemfile.
```

Comment:
```suggestion
It is possible to configure AppProfiler to use the [`vernier`](https://github.com/jhawthorn/vernier) or [`stackprof`](https://github.com/tmm1/stackprof). To use `vernier`, it must be added separately in the application Gemfile.
```

I wonder if we should be making stackprof a peer dependency as well (not in the gemspec, but required if defined in the bundle like vernier is). Maybe we can do that in a followup.